slim/train_image_classifier.py

```diff
@@ -457,12 +457,21 @@ def clone_fn(batch_queue):
       # Specify the loss function #
       #############################
       if 'AuxLogits' in end_points:
-        tf.losses.softmax_cross_entropy(
-            logits=end_points['AuxLogits'], onehot_labels=labels,
-            label_smoothing=FLAGS.label_smoothing, weights=0.4, scope='aux_loss')
+            logits = end_points['AuxLogits']
+            tf.losses.softmax_cross_entropy(
+                logits=logits, onehot_labels=labels,
+                label_smoothing=FLAGS.label_smoothing, weights=0.4, scope='aux_loss')
       tf.losses.softmax_cross_entropy(
-          logits=logits, onehot_labels=labels,
-          label_smoothing=FLAGS.label_smoothing, weights=1.0)
+            logits=logits, onehot_labels=labels,
+            label_smoothing=FLAGS.label_smoothing, weights=1.0)
+
+      #############################
+      ## Calculation of accuracy ##
+      #############################
+      accuracy = slim.metrics.accuracy(tf.to_int32(tf.argmax(logits, 1)),
+                                         tf.to_int32(tf.argmax(labels, 1)))
```

Comment:
I tested that ... Looks like it's not necessary but I think eliminating  tf.to_int32 caused me some issue  ... For now I remove it though!